2021-04-17 17:45:17,571 DEBUG log_decorator.py @log -> function parse_server_msg ran with args: ({'response': 200},) and kwargs: {}. /From module: client
2021-04-17 17:45:17,587 DEBUG log_decorator.py @log -> function parse_server_msg ran with args: ({'response': 400, 'error': 'Bad Request'},) and kwargs: {}. /From module: client
2021-04-17 17:45:17,587 DEBUG log_decorator.py @log -> function parse_server_msg ran with args: ({'error': 'Bad Request'},) and kwargs: {}. /From module: client
2021-04-17 17:45:17,602 DEBUG log_decorator.py @log -> function create_presence_msg ran with args: () and kwargs: {}. /From module: client
2021-04-17 17:54:23,065 DEBUG log_decorator.py @log -> function create_presence_msg ran with args: () and kwargs: {}. /From module: client
2021-04-17 17:55:18,698 DEBUG log_decorator.py @log -> function create_presence_msg ran with args: ('Guest',) and kwargs: {}. /From module: client
2021-04-17 17:56:56,511 DEBUG log_decorator.py @log -> function create_presence_msg ran with args: ('Guest',) and kwargs: {}. /From module: client
2021-04-17 17:58:08,506 DEBUG log_decorator.py @log -> function create_presence_msg ran with args: ('Guest',) and kwargs: {}. /From module: client
2021-04-17 17:58:26,480 DEBUG log_decorator.py @log -> function create_presence_msg ran with args: ('account_name',) and kwargs: {}. /From module: client
2021-04-17 17:58:32,877 DEBUG log_decorator.py @log -> function create_presence_msg ran with args: ('account_name',) and kwargs: {}. /From module: client
2021-04-17 17:59:37,700 DEBUG log_decorator.py @log -> function create_presence_msg ran with args: ('test_user',) and kwargs: {}. /From module: client
2021-04-17 17:59:58,932 DEBUG log_decorator.py @log -> function create_presence_msg ran with args: ('test_user',) and kwargs: {}. /From module: client
2021-04-17 18:01:22,642 DEBUG log_decorator.py @log -> function create_presence_msg ran with args: ('test_user',) and kwargs: {}. /From module: client
2021-04-17 18:01:49,537 DEBUG log_decorator.py @log -> function parse_server_msg ran with args: ({'response': 200},) and kwargs: {}. /From module: client
2021-04-17 18:01:49,537 DEBUG log_decorator.py @log -> function create_presence_msg ran with args: ('test_user',) and kwargs: {}. /From module: client
2021-04-17 18:05:38,175 DEBUG log_decorator.py @log -> function parse_server_msg ran with args: ('test_user',) and kwargs: {}. /From module: client
2021-04-17 18:05:38,190 DEBUG log_decorator.py @log -> function create_presence_msg ran with args: ('test_user',) and kwargs: {}. /From module: client
2021-04-17 18:07:07,808 DEBUG log_decorator.py @log -> function parse_server_msg ran with args: ('test_user',) and kwargs: {}. /From module: client
2021-04-17 18:07:07,808 DEBUG log_decorator.py @log -> function create_presence_msg ran with args: ('test_user',) and kwargs: {}. /From module: client
2021-04-17 18:19:32,637 DEBUG log_decorator.py @log -> function parse_server_msg ran with args: ({'response': 400}, 'test_user') and kwargs: {}. /From module: client
2021-04-17 18:19:32,651 DEBUG log_decorator.py @log -> function create_presence_msg ran with args: ('test_user',) and kwargs: {}. /From module: client
2021-04-17 18:19:58,841 DEBUG log_decorator.py @log -> function parse_server_msg ran with args: ({'response': 200}, 'test_user') and kwargs: {}. /From module: client
2021-04-17 18:19:58,850 DEBUG log_decorator.py @log -> function create_presence_msg ran with args: ('test_user',) and kwargs: {}. /From module: client
2021-04-17 18:21:37,343 DEBUG log_decorator.py @log -> function parse_cmd_arguments ran with args: () and kwargs: {}. /From module: __main__
2021-04-17 18:21:37,348 INFO server.py Запущен сервер, порт для подключений: , адрес с которого принимаются подключения: 5555. Если адрес не указан, принимаются соединения с любых адресов.
2021-04-17 18:21:42,891 INFO server.py Соедение с клиентом ('127.0.0.1', 50686) установлено
2021-04-17 18:21:42,896 DEBUG log_decorator.py @log -> function parse_client_msg ran with args: ({'action': 'presence', 'time': 1618672902, 'user': {'account_name': 'User1'}}, [], <socket.socket fd=156, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 5555), raddr=('127.0.0.1', 50686)>, [<socket.socket fd=156, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 5555), raddr=('127.0.0.1', 50686)>], {}) and kwargs: {}. /From module: __main__
2021-04-17 18:21:42,896 DEBUG server.py Разбор сообщения от клиента: {'action': 'presence', 'time': 1618672902, 'user': {'account_name': 'User1'}}
2021-04-17 18:23:00,579 DEBUG log_decorator.py @log -> function parse_server_msg ran with args: ({'response': 200}, 'test_user') and kwargs: {}. /From module: client
2021-04-17 18:23:00,590 DEBUG log_decorator.py @log -> function create_presence_msg ran with args: ('test_user',) and kwargs: {}. /From module: client
2021-04-17 18:23:20,378 ERROR server.py Клиент отключился от сервера. Тип исключения: ConnectionResetError, аргументы: (10054, 'Удаленный хост принудительно разорвал существующее подключение', None, 10054, None)
2021-04-17 18:24:23,894 DEBUG log_decorator.py @log -> function parse_cmd_arguments ran with args: () and kwargs: {}. /From module: __main__
2021-04-17 18:24:23,897 INFO server.py Запущен сервер, порт для подключений: , адрес с которого принимаются подключения: 5555. Если адрес не указан, принимаются соединения с любых адресов.
2021-04-17 18:24:27,708 INFO server.py Соедение с клиентом ('127.0.0.1', 50693) установлено
2021-04-17 18:24:27,908 DEBUG log_decorator.py @log -> function parse_client_msg ran with args: ({'action': 'presence', 'time': 1618673067, 'user': {'account_name': 'User1'}}, [], <socket.socket fd=156, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 5555), raddr=('127.0.0.1', 50693)>, [<socket.socket fd=156, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 5555), raddr=('127.0.0.1', 50693)>], {}) and kwargs: {}. /From module: __main__
2021-04-17 18:24:27,909 DEBUG server.py Разбор сообщения от клиента: {'action': 'presence', 'time': 1618673067, 'user': {'account_name': 'User1'}}
2021-04-17 18:52:00,616 ERROR server.py Клиент отключился от сервера. Тип исключения: ConnectionResetError, аргументы: (10054, 'Удаленный хост принудительно разорвал существующее подключение', None, 10054, None)
2021-04-17 18:52:04,960 DEBUG log_decorator.py @log -> function parse_cmd_arguments ran with args: () and kwargs: {}. /From module: __main__
2021-04-17 18:52:04,966 INFO server.py Запущен сервер, порт для подключений: , адрес с которого принимаются подключения: 5555. Если адрес не указан, принимаются соединения с любых адресов.
2021-04-17 18:52:08,749 INFO server.py Соедение с клиентом ('127.0.0.1', 50793) установлено
2021-04-17 18:52:08,949 ERROR server.py Клиент отключился от сервера. Тип исключения: JSONDecodeError, аргументы: ('Expecting value: line 1 column 1 (char 0)',)
2021-04-17 18:55:10,183 INFO server.py Соедение с клиентом ('127.0.0.1', 50800) установлено
2021-04-17 18:55:10,383 ERROR server.py Клиент отключился от сервера. Тип исключения: JSONDecodeError, аргументы: ('Expecting value: line 1 column 1 (char 0)',)
2021-04-17 19:01:48,813 INFO server.py Соедение с клиентом ('127.0.0.1', 50824) установлено
2021-04-17 19:01:48,814 DEBUG log_decorator.py @log -> function parse_client_msg ran with args: ({'action': 'presence', 'time': 1618675308, 'type': 'status', 'user': {'account_name': 'User1', 'account_password': 'password'}}, [], <socket.socket fd=156, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 5555), raddr=('127.0.0.1', 50824)>, [<socket.socket fd=156, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 5555), raddr=('127.0.0.1', 50824)>], {}) and kwargs: {}. /From module: __main__
2021-04-17 19:01:48,814 DEBUG server.py Разбор сообщения от клиента: {'action': 'presence', 'time': 1618675308, 'type': 'status', 'user': {'account_name': 'User1', 'account_password': 'password'}}
2021-04-17 19:01:58,615 ERROR server.py Клиент отключился от сервера. Тип исключения: ConnectionResetError, аргументы: (10054, 'Удаленный хост принудительно разорвал существующее подключение', None, 10054, None)
2021-04-17 19:02:03,494 DEBUG log_decorator.py @log -> function parse_cmd_arguments ran with args: () and kwargs: {}. /From module: __main__
2021-04-17 19:02:03,498 INFO server.py Запущен сервер, порт для подключений: , адрес с которого принимаются подключения: 5555. Если адрес не указан, принимаются соединения с любых адресов.
2021-04-17 19:02:09,594 INFO server.py Соедение с клиентом ('127.0.0.1', 50829) установлено
2021-04-17 19:02:09,595 DEBUG log_decorator.py @log -> function parse_client_msg ran with args: ({'action': 'presence', 'time': 1618675329, 'type': 'status', 'user': {'account_name': 'User1', 'account_password': 'password'}}, [], <socket.socket fd=156, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 5555), raddr=('127.0.0.1', 50829)>, [<socket.socket fd=156, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 5555), raddr=('127.0.0.1', 50829)>], {}) and kwargs: {}. /From module: __main__
2021-04-17 19:02:09,596 DEBUG server.py Разбор сообщения от клиента: {'action': 'presence', 'time': 1618675329, 'type': 'status', 'user': {'account_name': 'User1', 'account_password': 'password'}}
2021-04-17 19:02:28,413 DEBUG log_decorator.py @log -> function parse_client_msg ran with args: ({'action': 'message', 'time': 1618675348, 'from': 'User1', 'to': 'User1', 'message_text': 'Привет!'}, [], <socket.socket fd=156, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 5555), raddr=('127.0.0.1', 50829)>, [<socket.socket fd=156, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 5555), raddr=('127.0.0.1', 50829)>], {'User1': <socket.socket fd=156, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 5555), raddr=('127.0.0.1', 50829)>}) and kwargs: {}. /From module: __main__
2021-04-17 19:02:28,414 DEBUG server.py Разбор сообщения от клиента: {'action': 'message', 'time': 1618675348, 'from': 'User1', 'to': 'User1', 'message_text': 'Привет!'}
2021-04-17 19:02:28,414 DEBUG log_decorator.py @log -> function route_client_msg ran with args: ({'action': 'message', 'time': 1618675348, 'from': 'User1', 'to': 'User1', 'message_text': 'Привет!'}, {'User1': <socket.socket fd=156, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 5555), raddr=('127.0.0.1', 50829)>}, [<socket.socket fd=156, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 5555), raddr=('127.0.0.1', 50829)>]) and kwargs: {}. /From module: __main__
2021-04-17 19:02:28,415 INFO server.py Отправлено сообщение пользователю User1 от пользователя User1.
2021-04-17 19:08:45,427 ERROR server.py Клиент отключился от сервера. Тип исключения: ConnectionResetError, аргументы: (10054, 'Удаленный хост принудительно разорвал существующее подключение', None, 10054, None)
2021-04-17 19:08:49,733 DEBUG log_decorator.py @log -> function parse_cmd_arguments ran with args: () and kwargs: {}. /From module: __main__
2021-04-17 19:08:49,737 INFO server.py Запущен сервер, порт для подключений: , адрес с которого принимаются подключения: 5555. Если адрес не указан, принимаются соединения с любых адресов.
2021-04-17 19:08:53,520 INFO server.py Соедение с клиентом ('127.0.0.1', 50848) установлено
2021-04-17 19:08:53,521 DEBUG log_decorator.py @log -> function parse_client_msg ran with args: ({'action': 'presence', 'time': 1618675733, 'type': 'status', 'user': {'account_name': 'User1', 'account_password': 'password'}}, [], <socket.socket fd=156, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 5555), raddr=('127.0.0.1', 50848)>, [<socket.socket fd=156, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 5555), raddr=('127.0.0.1', 50848)>], {}) and kwargs: {}. /From module: __main__
2021-04-17 19:08:53,521 DEBUG server.py Разбор сообщения от клиента: {'action': 'presence', 'time': 1618675733, 'type': 'status', 'user': {'account_name': 'User1', 'account_password': 'password'}}
2021-04-17 19:08:54,738 ERROR server.py Клиент отключился от сервера. Тип исключения: ConnectionResetError, аргументы: (10054, 'Удаленный хост принудительно разорвал существующее подключение', None, 10054, None)
2021-04-17 19:11:25,070 INFO server.py Соедение с клиентом ('127.0.0.1', 50858) установлено
2021-04-17 19:11:25,271 DEBUG log_decorator.py @log -> function parse_client_msg ran with args: ({'action': 'presence', 'time': 1618675885, 'type': 'status', 'user': {'account_name': 'User1', 'account_password': 'password'}}, [], <socket.socket fd=220, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 5555), raddr=('127.0.0.1', 50858)>, [<socket.socket fd=220, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 5555), raddr=('127.0.0.1', 50858)>], {'User1': <socket.socket fd=156, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 5555), raddr=('127.0.0.1', 50848)>}) and kwargs: {}. /From module: __main__
2021-04-17 19:11:25,272 DEBUG server.py Разбор сообщения от клиента: {'action': 'presence', 'time': 1618675885, 'type': 'status', 'user': {'account_name': 'User1', 'account_password': 'password'}}
2021-04-17 19:11:33,348 DEBUG log_decorator.py @log -> function parse_cmd_arguments ran with args: () and kwargs: {}. /From module: __main__
2021-04-17 19:11:33,352 INFO server.py Запущен сервер, порт для подключений: , адрес с которого принимаются подключения: 5555. Если адрес не указан, принимаются соединения с любых адресов.
2021-04-17 19:11:37,450 INFO server.py Соедение с клиентом ('127.0.0.1', 50863) установлено
2021-04-17 19:11:37,650 DEBUG log_decorator.py @log -> function parse_client_msg ran with args: ({'action': 'presence', 'time': 1618675897, 'type': 'status', 'user': {'account_name': 'User1', 'account_password': 'password'}}, [], <socket.socket fd=156, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 5555), raddr=('127.0.0.1', 50863)>, [<socket.socket fd=156, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 5555), raddr=('127.0.0.1', 50863)>], {}) and kwargs: {}. /From module: __main__
2021-04-17 19:11:37,651 DEBUG server.py Разбор сообщения от клиента: {'action': 'presence', 'time': 1618675897, 'type': 'status', 'user': {'account_name': 'User1', 'account_password': 'password'}}
2021-04-17 19:11:50,253 DEBUG log_decorator.py @log -> function parse_client_msg ran with args: ({'action': 'message', 'time': 1618675910, 'from': 'User1', 'to': 'User1', 'message_text': 'Привет!'}, [], <socket.socket fd=156, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 5555), raddr=('127.0.0.1', 50863)>, [<socket.socket fd=156, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 5555), raddr=('127.0.0.1', 50863)>], {'User1': <socket.socket fd=156, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 5555), raddr=('127.0.0.1', 50863)>}) and kwargs: {}. /From module: __main__
2021-04-17 19:11:50,254 DEBUG server.py Разбор сообщения от клиента: {'action': 'message', 'time': 1618675910, 'from': 'User1', 'to': 'User1', 'message_text': 'Привет!'}
2021-04-17 19:11:50,254 DEBUG log_decorator.py @log -> function route_client_msg ran with args: ({'action': 'message', 'time': 1618675910, 'from': 'User1', 'to': 'User1', 'message_text': 'Привет!'}, {'User1': <socket.socket fd=156, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 5555), raddr=('127.0.0.1', 50863)>}, [<socket.socket fd=156, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 5555), raddr=('127.0.0.1', 50863)>]) and kwargs: {}. /From module: __main__
2021-04-17 19:11:50,254 INFO server.py Отправлено сообщение пользователю User1 от пользователя User1.
2021-04-17 19:27:49,876 DEBUG log_decorator.py @log -> function parse_client_msg ran with args: ({'action': 'presence', 'time': 1618676869, 'type': 'status', 'user': {'account_name': 'User1', 'account_password': 'password'}}, [], <socket.socket fd=156, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 5555), raddr=('127.0.0.1', 50863)>, [<socket.socket fd=156, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 5555), raddr=('127.0.0.1', 50863)>], {'User1': <socket.socket fd=156, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 5555), raddr=('127.0.0.1', 50863)>}) and kwargs: {}. /From module: __main__
2021-04-17 19:27:49,877 DEBUG server.py Разбор сообщения от клиента: {'action': 'presence', 'time': 1618676869, 'type': 'status', 'user': {'account_name': 'User1', 'account_password': 'password'}}
2021-04-17 19:28:23,006 DEBUG log_decorator.py @log -> function parse_cmd_arguments ran with args: () and kwargs: {}. /From module: __main__
2021-04-17 19:28:23,013 INFO server.py Запущен сервер, порт для подключений: , адрес с которого принимаются подключения: 5555. Если адрес не указан, принимаются соединения с любых адресов.
2021-04-17 19:28:29,113 INFO server.py Соедение с клиентом ('127.0.0.1', 50907) установлено
2021-04-17 19:28:29,314 ERROR server.py Клиент отключился от сервера. Тип исключения: ValueError, аргументы: ()
2021-04-17 19:56:41,604 DEBUG log_decorator.py @log -> function parse_cmd_arguments ran with args: () and kwargs: {}. /From module: __main__
2021-04-17 19:56:41,612 INFO server.py Запущен сервер, порт для подключений: , адрес с которого принимаются подключения: 5555. Если адрес не указан, принимаются соединения с любых адресов.
2021-04-17 19:56:47,986 INFO server.py Соедение с клиентом ('127.0.0.1', 50994) установлено
2021-04-17 19:58:58,720 DEBUG log_decorator.py @log -> function parse_cmd_arguments ran with args: () and kwargs: {}. /From module: __main__
2021-04-17 19:58:58,726 INFO server.py Запущен сервер, порт для подключений: , адрес с которого принимаются подключения: 5555. Если адрес не указан, принимаются соединения с любых адресов.
2021-04-17 19:59:02,571 INFO server.py Соедение с клиентом ('127.0.0.1', 51063) установлено
2021-04-17 20:00:58,968 DEBUG log_decorator.py @log -> function parse_cmd_arguments ran with args: () and kwargs: {}. /From module: __main__
2021-04-17 20:00:58,974 INFO server.py Запущен сервер, порт для подключений: , адрес с которого принимаются подключения: 5555. Если адрес не указан, принимаются соединения с любых адресов.
2021-04-17 20:01:03,723 INFO server.py Соедение с клиентом ('127.0.0.1', 51079) установлено
2021-04-17 20:01:03,928 DEBUG log_decorator.py @log -> function parse_client_msg ran with args: ({'action': 'presence', 'time': 1618678863, 'type': 'status', 'user': {'account_name': 'User1', 'account_password': 'password'}}, [], <socket.socket fd=156, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 5555), raddr=('127.0.0.1', 51079)>, [<socket.socket fd=156, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 5555), raddr=('127.0.0.1', 51079)>], {}) and kwargs: {}. /From module: __main__
2021-04-17 20:01:03,928 DEBUG server.py Разбор сообщения от клиента: {'action': 'presence', 'time': 1618678863, 'type': 'status', 'user': {'account_name': 'User1', 'account_password': 'password'}}
2021-04-17 20:04:30,939 ERROR server.py Клиент отключился от сервера. Тип исключения: ConnectionResetError, аргументы: (10054, 'Удаленный хост принудительно разорвал существующее подключение', None, 10054, None)
2021-04-17 20:04:34,113 DEBUG log_decorator.py @log -> function parse_cmd_arguments ran with args: () and kwargs: {}. /From module: __main__
2021-04-17 20:04:34,120 INFO server.py Запущен сервер, порт для подключений: , адрес с которого принимаются подключения: 5555. Если адрес не указан, принимаются соединения с любых адресов.
2021-04-17 20:04:41,808 INFO server.py Соедение с клиентом ('127.0.0.1', 51107) установлено
2021-04-17 20:08:52,388 DEBUG log_decorator.py @log -> function parse_cmd_arguments ran with args: () and kwargs: {}. /From module: __main__
2021-04-17 20:08:52,394 INFO server.py Запущен сервер, порт для подключений: , адрес с которого принимаются подключения: 5555. Если адрес не указан, принимаются соединения с любых адресов.
2021-04-17 20:08:58,037 INFO server.py Соедение с клиентом ('127.0.0.1', 51120) установлено
2021-04-17 20:08:58,038 DEBUG log_decorator.py @log -> function parse_client_msg ran with args: ({'action': 'presence', 'time': 1618679338, 'type': 'status', 'user': {'account_name': 'User1', 'account_password': 'password'}}, [], <socket.socket fd=156, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 5555), raddr=('127.0.0.1', 51120)>, [<socket.socket fd=156, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 5555), raddr=('127.0.0.1', 51120)>], {}) and kwargs: {}. /From module: __main__
2021-04-17 20:08:58,038 DEBUG server.py Разбор сообщения от клиента: {'action': 'presence', 'time': 1618679338, 'type': 'status', 'user': {'account_name': 'User1', 'account_password': 'password'}}
2021-04-17 20:09:34,445 DEBUG log_decorator.py @log -> function parse_client_msg ran with args: ({'action': 'message', 'time': 1618679374, 'from': 'User1', 'to': 'User1', 'message_text': 'Привет!'}, [], <socket.socket fd=156, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 5555), raddr=('127.0.0.1', 51120)>, [<socket.socket fd=156, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 5555), raddr=('127.0.0.1', 51120)>], {'User1': <socket.socket fd=156, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 5555), raddr=('127.0.0.1', 51120)>}) and kwargs: {}. /From module: __main__
2021-04-17 20:09:34,446 DEBUG server.py Разбор сообщения от клиента: {'action': 'message', 'time': 1618679374, 'from': 'User1', 'to': 'User1', 'message_text': 'Привет!'}
2021-04-17 20:09:34,446 DEBUG log_decorator.py @log -> function route_client_msg ran with args: ({'action': 'message', 'time': 1618679374, 'from': 'User1', 'to': 'User1', 'message_text': 'Привет!'}, {'User1': <socket.socket fd=156, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 5555), raddr=('127.0.0.1', 51120)>}, [<socket.socket fd=156, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 5555), raddr=('127.0.0.1', 51120)>]) and kwargs: {}. /From module: __main__
2021-04-17 20:09:34,446 INFO server.py Отправлено сообщение пользователю User1 от пользователя User1.
2021-04-17 20:24:08,442 DEBUG log_decorator.py @log -> function parse_client_msg ran with args: ({'action': 'presence', 'time': 1618680248, 'type': 'status', 'user': {'account_name': 'User1', 'account_password': 'password'}}, [], <socket.socket fd=156, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 5555), raddr=('127.0.0.1', 51120)>, [<socket.socket fd=156, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 5555), raddr=('127.0.0.1', 51120)>], {'User1': <socket.socket fd=156, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 5555), raddr=('127.0.0.1', 51120)>}) and kwargs: {}. /From module: __main__
2021-04-17 20:24:08,443 DEBUG server.py Разбор сообщения от клиента: {'action': 'presence', 'time': 1618680248, 'type': 'status', 'user': {'account_name': 'User1', 'account_password': 'password'}}
2021-04-17 20:24:23,815 DEBUG log_decorator.py @log -> function parse_cmd_arguments ran with args: () and kwargs: {}. /From module: __main__
2021-04-17 20:24:23,821 INFO server.py Запущен сервер, порт для подключений: , адрес с которого принимаются подключения: 5555. Если адрес не указан, принимаются соединения с любых адресов.
2021-04-17 20:24:32,684 INFO server.py Соедение с клиентом ('127.0.0.1', 51169) установлено
2021-04-17 20:24:32,685 DEBUG log_decorator.py @log -> function parse_client_msg ran with args: ({'action': 'presence', 'time': 1618680272, 'type': 'status', 'user': {'account_name': 'User1', 'account_password': 'password'}}, [], <socket.socket fd=156, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 5555), raddr=('127.0.0.1', 51169)>, [<socket.socket fd=156, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 5555), raddr=('127.0.0.1', 51169)>], {}) and kwargs: {}. /From module: __main__
2021-04-17 20:24:32,686 DEBUG server.py Разбор сообщения от клиента: {'action': 'presence', 'time': 1618680272, 'type': 'status', 'user': {'account_name': 'User1', 'account_password': 'password'}}
2021-04-17 20:27:00,350 ERROR server.py Клиент отключился от сервера. Тип исключения: ConnectionResetError, аргументы: (10054, 'Удаленный хост принудительно разорвал существующее подключение', None, 10054, None)
2021-04-17 20:27:01,043 INFO server.py Соедение с клиентом ('127.0.0.1', 51179) установлено
2021-04-17 20:27:01,244 ERROR server.py Клиент отключился от сервера. Тип исключения: ValueError, аргументы: ()
2021-04-17 20:27:09,674 DEBUG log_decorator.py @log -> function parse_cmd_arguments ran with args: () and kwargs: {}. /From module: __main__
2021-04-17 20:27:09,678 INFO server.py Запущен сервер, порт для подключений: , адрес с которого принимаются подключения: 5555. Если адрес не указан, принимаются соединения с любых адресов.
2021-04-17 20:27:12,671 INFO server.py Соедение с клиентом ('127.0.0.1', 51180) установлено
2021-04-17 20:27:12,871 ERROR server.py Клиент отключился от сервера. Тип исключения: ValueError, аргументы: ()
2021-04-17 20:28:33,791 DEBUG log_decorator.py @log -> function parse_cmd_arguments ran with args: () and kwargs: {}. /From module: __main__
2021-04-17 20:28:33,796 INFO server.py Запущен сервер, порт для подключений: , адрес с которого принимаются подключения: 5555. Если адрес не указан, принимаются соединения с любых адресов.
2021-04-17 20:28:36,628 INFO server.py Соедение с клиентом ('127.0.0.1', 51183) установлено
2021-04-17 20:28:36,835 DEBUG log_decorator.py @log -> function parse_client_msg ran with args: ({'action': 'presence', 'time': 1618680516, 'type': 'status', 'user': {'account_name': 'User1', 'account_password': 'password'}}, [], <socket.socket fd=156, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 5555), raddr=('127.0.0.1', 51183)>, [<socket.socket fd=156, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 5555), raddr=('127.0.0.1', 51183)>], {}) and kwargs: {}. /From module: __main__
2021-04-17 20:28:36,836 DEBUG server.py Разбор сообщения от клиента: {'action': 'presence', 'time': 1618680516, 'type': 'status', 'user': {'account_name': 'User1', 'account_password': 'password'}}
2021-04-17 20:29:23,396 DEBUG log_decorator.py @log -> function parse_cmd_arguments ran with args: () and kwargs: {}. /From module: __main__
2021-04-17 20:29:23,401 INFO server.py Запущен сервер, порт для подключений: , адрес с которого принимаются подключения: 5555. Если адрес не указан, принимаются соединения с любых адресов.
2021-04-17 20:29:26,056 INFO server.py Соедение с клиентом ('127.0.0.1', 51189) установлено
2021-04-17 20:29:26,057 DEBUG log_decorator.py @log -> function parse_client_msg ran with args: ({'action': 'presence', 'time': 1618680566, 'type': 'status', 'user': {'account_name': 'User1', 'account_password': 'password'}}, [], <socket.socket fd=156, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 5555), raddr=('127.0.0.1', 51189)>, [<socket.socket fd=156, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 5555), raddr=('127.0.0.1', 51189)>], {}) and kwargs: {}. /From module: __main__
2021-04-17 20:29:26,058 DEBUG server.py Разбор сообщения от клиента: {'action': 'presence', 'time': 1618680566, 'type': 'status', 'user': {'account_name': 'User1', 'account_password': 'password'}}
2021-04-17 20:29:53,992 DEBUG log_decorator.py @log -> function parse_cmd_arguments ran with args: () and kwargs: {}. /From module: __main__
2021-04-17 20:29:53,998 INFO server.py Запущен сервер, порт для подключений: , адрес с которого принимаются подключения: 5555. Если адрес не указан, принимаются соединения с любых адресов.
2021-04-17 20:29:55,709 INFO server.py Соедение с клиентом ('127.0.0.1', 51192) установлено
2021-04-17 20:29:55,910 DEBUG log_decorator.py @log -> function parse_client_msg ran with args: ({'action': 'presence', 'time': 1618680595, 'type': 'status', 'user': {'account_name': 'User1', 'account_password': 'password'}}, [], <socket.socket fd=156, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 5555), raddr=('127.0.0.1', 51192)>, [<socket.socket fd=156, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 5555), raddr=('127.0.0.1', 51192)>], {}) and kwargs: {}. /From module: __main__
2021-04-17 20:29:55,911 DEBUG server.py Разбор сообщения от клиента: {'action': 'presence', 'time': 1618680595, 'type': 'status', 'user': {'account_name': 'User1', 'account_password': 'password'}}
2021-04-17 20:36:03,960 DEBUG log_decorator.py @log -> function parse_cmd_arguments ran with args: () and kwargs: {}. /From module: __main__
2021-04-17 20:36:03,965 INFO server.py Запущен сервер, порт для подключений: , адрес с которого принимаются подключения: 5555. Если адрес не указан, принимаются соединения с любых адресов.
2021-04-17 20:36:09,583 INFO server.py Соедение с клиентом ('127.0.0.1', 51225) установлено
2021-04-17 20:36:09,784 DEBUG log_decorator.py @log -> function parse_client_msg ran with args: ({'action': 'presence', 'time': 1618680969, 'type': 'status', 'user': {'account_name': 'User1', 'account_password': 'password'}}, [], <socket.socket fd=156, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 5555), raddr=('127.0.0.1', 51225)>, [<socket.socket fd=156, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 5555), raddr=('127.0.0.1', 51225)>], {}) and kwargs: {}. /From module: __main__
2021-04-17 20:36:09,784 DEBUG server.py Разбор сообщения от клиента: {'action': 'presence', 'time': 1618680969, 'type': 'status', 'user': {'account_name': 'User1', 'account_password': 'password'}}
2021-04-17 20:40:45,025 DEBUG log_decorator.py @log -> function parse_cmd_arguments ran with args: () and kwargs: {}. /From module: __main__
2021-04-17 20:40:45,031 INFO server.py Запущен сервер, порт для подключений: , адрес с которого принимаются подключения: 5555. Если адрес не указан, принимаются соединения с любых адресов.
2021-04-17 20:40:48,416 INFO server.py Соедение с клиентом ('127.0.0.1', 51238) установлено
2021-04-17 20:40:48,618 DEBUG log_decorator.py @log -> function parse_client_msg ran with args: ({'action': 'presence', 'time': 1618681248, 'type': 'status', 'user': {'account_name': 'User1', 'account_password': 'password'}}, [], <socket.socket fd=156, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 5555), raddr=('127.0.0.1', 51238)>, [<socket.socket fd=156, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 5555), raddr=('127.0.0.1', 51238)>], {}) and kwargs: {}. /From module: __main__
2021-04-17 20:40:48,619 DEBUG server.py Разбор сообщения от клиента: {'action': 'presence', 'time': 1618681248, 'type': 'status', 'user': {'account_name': 'User1', 'account_password': 'password'}}
2021-04-17 20:41:05,819 DEBUG log_decorator.py @log -> function parse_client_msg ran with args: ({'action': 'message', 'time': 1618681265, 'from': 'User1', 'to': 'User1', 'message_text': 'Привет!'}, [], <socket.socket fd=156, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 5555), raddr=('127.0.0.1', 51238)>, [<socket.socket fd=156, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 5555), raddr=('127.0.0.1', 51238)>], {'User1': <socket.socket fd=156, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 5555), raddr=('127.0.0.1', 51238)>}) and kwargs: {}. /From module: __main__
2021-04-17 20:41:05,820 DEBUG server.py Разбор сообщения от клиента: {'action': 'message', 'time': 1618681265, 'from': 'User1', 'to': 'User1', 'message_text': 'Привет!'}
2021-04-17 20:41:05,820 DEBUG log_decorator.py @log -> function route_client_msg ran with args: ({'action': 'message', 'time': 1618681265, 'from': 'User1', 'to': 'User1', 'message_text': 'Привет!'}, {'User1': <socket.socket fd=156, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 5555), raddr=('127.0.0.1', 51238)>}, [<socket.socket fd=156, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 5555), raddr=('127.0.0.1', 51238)>]) and kwargs: {}. /From module: __main__
2021-04-17 20:41:05,821 INFO server.py Отправлено сообщение пользователю User1 от пользователя User1.
2021-04-17 20:41:47,674 ERROR server.py Клиент отключился от сервера. Тип исключения: ConnectionResetError, аргументы: (10054, 'Удаленный хост принудительно разорвал существующее подключение', None, 10054, None)
2021-04-17 20:41:50,154 DEBUG log_decorator.py @log -> function parse_cmd_arguments ran with args: () and kwargs: {}. /From module: __main__
2021-04-17 20:41:50,158 INFO server.py Запущен сервер, порт для подключений: , адрес с которого принимаются подключения: 5555. Если адрес не указан, принимаются соединения с любых адресов.
2021-04-17 20:41:52,811 INFO server.py Соедение с клиентом ('127.0.0.1', 51247) установлено
2021-04-17 20:41:53,012 DEBUG log_decorator.py @log -> function parse_client_msg ran with args: ({'action': 'presence', 'time': 1618681312, 'type': 'status', 'user': {'account_name': 'User1', 'account_password': 'password'}}, [], <socket.socket fd=156, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 5555), raddr=('127.0.0.1', 51247)>, [<socket.socket fd=156, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 5555), raddr=('127.0.0.1', 51247)>], {}) and kwargs: {}. /From module: __main__
2021-04-17 20:41:53,013 DEBUG server.py Разбор сообщения от клиента: {'action': 'presence', 'time': 1618681312, 'type': 'status', 'user': {'account_name': 'User1', 'account_password': 'password'}}
2021-04-17 20:42:06,025 ERROR server.py Клиент отключился от сервера. Тип исключения: ConnectionResetError, аргументы: (10054, 'Удаленный хост принудительно разорвал существующее подключение', None, 10054, None)
2021-04-17 20:42:09,369 DEBUG log_decorator.py @log -> function parse_cmd_arguments ran with args: () and kwargs: {}. /From module: __main__
2021-04-17 20:42:09,373 INFO server.py Запущен сервер, порт для подключений: , адрес с которого принимаются подключения: 5555. Если адрес не указан, принимаются соединения с любых адресов.
2021-04-17 20:42:12,849 INFO server.py Соедение с клиентом ('127.0.0.1', 51249) установлено
2021-04-17 20:42:13,049 DEBUG log_decorator.py @log -> function parse_client_msg ran with args: ({'action': 'presence', 'time': 1618681332, 'type': 'status', 'user': {'account_name': 'User1', 'account_password': 'password'}}, [], <socket.socket fd=156, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 5555), raddr=('127.0.0.1', 51249)>, [<socket.socket fd=156, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 5555), raddr=('127.0.0.1', 51249)>], {}) and kwargs: {}. /From module: __main__
2021-04-17 20:42:13,049 DEBUG server.py Разбор сообщения от клиента: {'action': 'presence', 'time': 1618681332, 'type': 'status', 'user': {'account_name': 'User1', 'account_password': 'password'}}
2021-04-17 20:42:24,651 DEBUG log_decorator.py @log -> function parse_client_msg ran with args: ({'action': 'message', 'time': 1618681344, 'from': 'User1', 'to': 'User1', 'message_text': 'Привет!'}, [], <socket.socket fd=156, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 5555), raddr=('127.0.0.1', 51249)>, [<socket.socket fd=156, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 5555), raddr=('127.0.0.1', 51249)>], {'User1': <socket.socket fd=156, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 5555), raddr=('127.0.0.1', 51249)>}) and kwargs: {}. /From module: __main__
2021-04-17 20:42:24,651 DEBUG server.py Разбор сообщения от клиента: {'action': 'message', 'time': 1618681344, 'from': 'User1', 'to': 'User1', 'message_text': 'Привет!'}
2021-04-17 20:42:24,651 DEBUG log_decorator.py @log -> function route_client_msg ran with args: ({'action': 'message', 'time': 1618681344, 'from': 'User1', 'to': 'User1', 'message_text': 'Привет!'}, {'User1': <socket.socket fd=156, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 5555), raddr=('127.0.0.1', 51249)>}, [<socket.socket fd=156, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 5555), raddr=('127.0.0.1', 51249)>]) and kwargs: {}. /From module: __main__
2021-04-17 20:42:24,652 INFO server.py Отправлено сообщение пользователю User1 от пользователя User1.
2021-04-17 20:43:04,958 ERROR server.py Клиент отключился от сервера. Тип исключения: ConnectionResetError, аргументы: (10054, 'Удаленный хост принудительно разорвал существующее подключение', None, 10054, None)
2021-04-17 20:43:08,512 DEBUG log_decorator.py @log -> function parse_cmd_arguments ran with args: () and kwargs: {}. /From module: __main__
2021-04-17 20:43:08,517 INFO server.py Запущен сервер, порт для подключений: , адрес с которого принимаются подключения: 5555. Если адрес не указан, принимаются соединения с любых адресов.
2021-04-17 20:43:11,933 INFO server.py Соедение с клиентом ('127.0.0.1', 51254) установлено
2021-04-17 20:43:12,134 DEBUG log_decorator.py @log -> function parse_client_msg ran with args: ({'action': 'presence', 'time': 1618681391, 'type': 'status', 'user': {'account_name': 'User1', 'account_password': 'password'}}, [], <socket.socket fd=156, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 5555), raddr=('127.0.0.1', 51254)>, [<socket.socket fd=156, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 5555), raddr=('127.0.0.1', 51254)>], {}) and kwargs: {}. /From module: __main__
2021-04-17 20:43:12,135 DEBUG server.py Разбор сообщения от клиента: {'action': 'presence', 'time': 1618681391, 'type': 'status', 'user': {'account_name': 'User1', 'account_password': 'password'}}
2021-04-17 20:43:48,567 DEBUG log_decorator.py @log -> function parse_client_msg ran with args: ({'action': 'message', 'time': 1618681428, 'from': 'User1', 'to': 'User1', 'message_text': 'Привет!'}, [], <socket.socket fd=156, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 5555), raddr=('127.0.0.1', 51254)>, [<socket.socket fd=156, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 5555), raddr=('127.0.0.1', 51254)>], {'User1': <socket.socket fd=156, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 5555), raddr=('127.0.0.1', 51254)>}) and kwargs: {}. /From module: __main__
2021-04-17 20:43:48,567 DEBUG server.py Разбор сообщения от клиента: {'action': 'message', 'time': 1618681428, 'from': 'User1', 'to': 'User1', 'message_text': 'Привет!'}
2021-04-17 20:43:48,568 DEBUG log_decorator.py @log -> function route_client_msg ran with args: ({'action': 'message', 'time': 1618681428, 'from': 'User1', 'to': 'User1', 'message_text': 'Привет!'}, {'User1': <socket.socket fd=156, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 5555), raddr=('127.0.0.1', 51254)>}, [<socket.socket fd=156, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 5555), raddr=('127.0.0.1', 51254)>]) and kwargs: {}. /From module: __main__
2021-04-17 20:43:48,568 INFO server.py Отправлено сообщение пользователю User1 от пользователя User1.
2021-04-17 20:45:01,167 DEBUG log_decorator.py @log -> function parse_client_msg ran with args: ({'action': 'message', 'time': 1618681501, 'from': 'User1', 'to': 'User1', 'message_text': 'Кусь!'}, [], <socket.socket fd=156, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 5555), raddr=('127.0.0.1', 51254)>, [<socket.socket fd=156, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 5555), raddr=('127.0.0.1', 51254)>], {'User1': <socket.socket fd=156, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 5555), raddr=('127.0.0.1', 51254)>}) and kwargs: {}. /From module: __main__
2021-04-17 20:45:01,168 DEBUG server.py Разбор сообщения от клиента: {'action': 'message', 'time': 1618681501, 'from': 'User1', 'to': 'User1', 'message_text': 'Кусь!'}
2021-04-17 20:45:01,168 DEBUG log_decorator.py @log -> function route_client_msg ran with args: ({'action': 'message', 'time': 1618681501, 'from': 'User1', 'to': 'User1', 'message_text': 'Кусь!'}, {'User1': <socket.socket fd=156, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 5555), raddr=('127.0.0.1', 51254)>}, [<socket.socket fd=156, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 5555), raddr=('127.0.0.1', 51254)>]) and kwargs: {}. /From module: __main__
2021-04-17 20:45:01,169 INFO server.py Отправлено сообщение пользователю User1 от пользователя User1.
2021-04-17 20:45:54,501 ERROR server.py Клиент отключился от сервера. Тип исключения: ConnectionResetError, аргументы: (10054, 'Удаленный хост принудительно разорвал существующее подключение', None, 10054, None)
2021-04-17 20:53:02,611 DEBUG log_decorator.py @log -> function parse_cmd_arguments ran with args: () and kwargs: {}. /From module: __main__
2021-04-17 20:53:02,616 INFO server.py Запущен сервер, порт для подключений: , адрес с которого принимаются подключения: 5555. Если адрес не указан, принимаются соединения с любых адресов.
2021-04-17 20:53:12,989 INFO server.py Соедение с клиентом ('127.0.0.1', 51285) установлено
2021-04-17 20:53:13,190 DEBUG log_decorator.py @log -> function parse_client_msg ran with args: ({'action': 'presence', 'time': 1618681992, 'type': 'status', 'user': {'account_name': 'User1', 'account_password': 'password'}}, [], <socket.socket fd=160, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 5555), raddr=('127.0.0.1', 51285)>, [<socket.socket fd=160, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 5555), raddr=('127.0.0.1', 51285)>], {}) and kwargs: {}. /From module: __main__
2021-04-17 20:53:13,191 DEBUG server.py Разбор сообщения от клиента: {'action': 'presence', 'time': 1618681992, 'type': 'status', 'user': {'account_name': 'User1', 'account_password': 'password'}}
2021-04-17 20:53:15,122 INFO server.py Соедение с клиентом ('127.0.0.1', 51286) установлено
2021-04-17 20:53:15,322 DEBUG log_decorator.py @log -> function parse_client_msg ran with args: ({'action': 'presence', 'time': 1618681995, 'type': 'status', 'user': {'account_name': 'User2', 'account_password': 'password'}}, [], <socket.socket fd=224, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 5555), raddr=('127.0.0.1', 51286)>, [<socket.socket fd=160, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 5555), raddr=('127.0.0.1', 51285)>, <socket.socket fd=224, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 5555), raddr=('127.0.0.1', 51286)>], {'User1': <socket.socket fd=160, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 5555), raddr=('127.0.0.1', 51285)>}) and kwargs: {}. /From module: __main__
2021-04-17 20:53:15,322 DEBUG server.py Разбор сообщения от клиента: {'action': 'presence', 'time': 1618681995, 'type': 'status', 'user': {'account_name': 'User2', 'account_password': 'password'}}
2021-04-17 20:53:38,125 DEBUG log_decorator.py @log -> function parse_client_msg ran with args: ({'action': 'message', 'time': 1618682018, 'from': 'User1', 'to': 'User2', 'message_text': 'Привет, User2!'}, [], <socket.socket fd=160, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 5555), raddr=('127.0.0.1', 51285)>, [<socket.socket fd=160, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 5555), raddr=('127.0.0.1', 51285)>, <socket.socket fd=224, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 5555), raddr=('127.0.0.1', 51286)>], {'User1': <socket.socket fd=160, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 5555), raddr=('127.0.0.1', 51285)>, 'User2': <socket.socket fd=224, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 5555), raddr=('127.0.0.1', 51286)>}) and kwargs: {}. /From module: __main__
2021-04-17 20:53:38,126 DEBUG server.py Разбор сообщения от клиента: {'action': 'message', 'time': 1618682018, 'from': 'User1', 'to': 'User2', 'message_text': 'Привет, User2!'}
2021-04-17 20:53:38,126 DEBUG log_decorator.py @log -> function route_client_msg ran with args: ({'action': 'message', 'time': 1618682018, 'from': 'User1', 'to': 'User2', 'message_text': 'Привет, User2!'}, {'User1': <socket.socket fd=160, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 5555), raddr=('127.0.0.1', 51285)>, 'User2': <socket.socket fd=224, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 5555), raddr=('127.0.0.1', 51286)>}, [<socket.socket fd=160, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 5555), raddr=('127.0.0.1', 51285)>, <socket.socket fd=224, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 5555), raddr=('127.0.0.1', 51286)>]) and kwargs: {}. /From module: __main__
2021-04-17 20:53:38,127 INFO server.py Отправлено сообщение пользователю User2 от пользователя User1.
2021-04-17 20:55:48,332 ERROR server.py Клиент отключился от сервера. Тип исключения: ConnectionResetError, аргументы: (10054, 'Удаленный хост принудительно разорвал существующее подключение', None, 10054, None)
2021-04-17 20:55:50,136 ERROR server.py Клиент отключился от сервера. Тип исключения: ConnectionResetError, аргументы: (10054, 'Удаленный хост принудительно разорвал существующее подключение', None, 10054, None)
